dbtwiz/model/create.py

```diff
@@ -508,10 +509,9 @@ def create_model_files(
         with open(sql_path, "w+") as f:
             f.write(sql)
 
-        os.system(f"code {sql_path}")
-    os.system(f"code {yml_path}")
+        os.system(f"{user_config().editor} {sql_path}")
```

Comment:
Since we do this multiple places, maybe it should be a function instead for opening a file path in the user defined editor?

Reply:
Added in `helpers/editor.py`.